memfs/memory_test.go

```diff
@@ -85,6 +86,17 @@ func (s *MemorySuite) TestOrder(c *C) {
 	}
 }
 
+func (s *MemorySuite) TestNotFound(c *C) {
+	files, err := s.FS.ReadDir("asdf")
+	c.Assert(files, HasLen, 0)
+	// JS / wasip have this error message captalised.
+	msg := "open /asdf: (N|n)o such file or directory"
+	if runtime.GOOS == "windows" {
+		msg = "open \\asdf: The system cannot find the file specified."
```

Comment:
This is used as regex, so special chars need to be escaped.
```suggestion
		msg = "open \\\\asdf: The system cannot find the file specified\."
```